.github/workflows

```diff
@@ -0,0 +1,25 @@
+name: vcstool
```

Comment:
just in case you're wondering why the workflow isn't getting executed, this file has to be a `*.yml` file in the `.github/workflows/` directory

Reply:
Thanks. Good point. First try :wink: